README.md

```diff
@@ -225,21 +225,9 @@ epoch.  We define a tie-breaking rule that allows all remaining members to
 deterministically choose one of those tied epochs as the preferred epoch.  This
 rule will be used in sections 4.4. and 4.6.
 
-Suppose:
-
-- `keys` is the set of all epoch keys that are currently "tied"
-- `keys_count` is the size of the set `keys`
-- `sorted_keys` is the lexicographically sorted (by hexadecimal encoding) list
-  of all epoch keys in `keys`
-- `concat_sorted_keys` is the concatenation of all epoch keys in `sorted_keys`
-- `hashed` is the SHA-256 hash of `concat_sorted_keys`
-- `hashed8` are the first 8 bytes of `hashed`
-- `as_number` is the integer value of `hashed8` in big-endian byte order
-- `winning_index` is the remainder of `as_number` divided by `keys_count`, that
-  is, `as_number modulo keys_count`
-
-Then the tie-breaking rule is: pick the epoch key at index `winning_index` in
-`sorted_keys` as the winner.  List indexes are zero-based.
+The tie-breaking rule is: among the forked epochs, sort their epoch keys by
+lexicographic order (in hexadecimal encoding) and the epoch with the smallest
```

Comment:
Should we do base64 instead? That's what we use in all the other cases, e.g. when embedding a key in a message, or in the feed purpose

Reply:
true!